recirq/qcqmc/experiment_test.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2024 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Tuple
+
+import cirq
+
+from recirq.qcqmc.blueprint import BlueprintData
+from recirq.qcqmc.experiment import SimulatedExperimentParams, build_experiment
+from recirq.qcqmc.hamiltonian import HamiltonianData
+from recirq.qcqmc.trial_wf import TrialWavefunctionData
+
+
+def test_small_experiment_raw_samples_shape(
```

Comment:
This test file seems short.  Does it test everything in experiments?

Reply:
It does at least call every function (after a small update) but the test coverage is not great. I think I'd @wjhuggins to help writing some sensible physical tests of the shadow tomography protocol but I think that can be a follow up (we discussed writing some better documentation of the shadow tomography part of the code which is probably of most interest to QIS people).

Reply:
Ok, yeah, let's make it a follow up PR.